docs/changelog/134941.yaml

```
@@ -0,0 +1,6 @@
+pr: 134941
+summary: "Streams: Adjust `max_age` for tiny retentions"
```

Let's change this since it's not specific to streams:
```suggestion
summary: "DLM: Improve automatic `max_age` rollover condition for tiny retentions"
```

---

updated to best of my ability to keep title under 50 chars

---

We don't have to keep the summaries under 50 chars I think? That limit is usually for GH PR titles I believe.

---

ah, assumed we want to keep these the same

i'll keep this in mind for the future